pkg/network/tracer/ebpf_conntracker.go

```diff
@@ -470,9 +471,12 @@ func getManager(cfg *config.Config, buf io.ReaderAt, constants []manager.Constan
 }
 
 func getPrebuiltConntracker(cfg *config.Config) (bytecode.AssetReader, []manager.ConstantEditor, error) {
-	kv414 := kernel.VersionCode(4, 14, 0)
-	if kv, err := kernel.HostVersion(); err != nil || kv < kv414 {
-		return nil, nil, fmt.Errorf("ebpf conntracker requires kernel version %s or higher", kv414)
+	kv, err := ebpfkernel.NewKernelVersion()
+	if err != nil {
+		return nil, nil, fmt.Errorf("unable to detect the kernel version: %s", err)
+	}
+	if kv.Code < ebpfkernel.Kernel4_14 && !kv.IsRH7Kernel() {
+		return nil, nil, fmt.Errorf("ebpf conntracker requires kernel version 4.14 or higher or a RHEL kernel with backported eBPF support")
```

Comment:
I think if you make this code block a helper you can call it here and in the tests to relieve some of the copy-pasting.